src/main/scala/no/gdl/bookapi/service/FeedService.scala

```diff
@@ -40,7 +40,7 @@ trait FeedService {
           api.FeedDefinition(
             feedDefinition.id.get,
             feedDefinition.revision.get,
-            s"${BookApiProperties.Domain}${feedDefinition.url}",
+            s"${BookApiProperties.CloudFrontOpds}${feedDefinition.url.replace(BookApiProperties.OpdsPath,"")}",
```

Comment:
FeedDefinition trenger OpdsPath for at dei skal genereres på korrekt url, men i og med at CloudFrontOpds tar høgde for verdien må den vekk før self-url defineres.